Palo Alto Networks/paloalto-ngfw/_meta/smart-descriptions.json

```diff
@@ -29,26 +29,27 @@
             {
                 "field": "rule.name"
             }
-        ]
-    },
-    {
-        "value": "{user.name} on {host.name} was classified as {paloalto.PanOSQuarantineReason} because of {event.reason}",
-        "conditions": [
-            {
-                "field": "log.logger",
-                "value": "globalprotect"
-            },
+        ],
+        "relationships": [
             {
-                "field": "user.name"
+                "source": "source.ip",
+                "target": "destination.ip",
+                "type": "connected to"
             },
             {
-                "field": "host.name"
+                "source": "user.name",
+                "target": "source.ip",
+                "type": "connected from"
             },
             {
-                "field": "paloalto.PanOSQuarantineReason"
+                "source": "destination.ip",
+                "target": "host.name",
+                "type": "associated with"
             },
             {
-                "field": "event.reason"
+                "source": "process.name",
+                "target": "running on",
+                "type": "source.ip"
```

Comment:
Just a question: why `target` and `type` are invert (regarding the other field above) ? 🤔

Reply:
Nice catch ! It should be fixed in latest commit